### ISSUE TITLE
CHANGELOG

### ISSUE THREAD

**Issue by joshsmith** [coderly/process#1]
I propose we have a `CHANGELOG.md` file that lays out what's changed in our process. At the start of a given week we can send this out to everyone, as well, so we can keep track of what things have happened.

I don't want to use email as a tool for this because it'll be hard to keep track. I get too much email as it is. So I don't want to rely on GH notifications.

We should be making an attempt to summarize and link to changes, anyway.

**Comment by venkatd** [comment]
I agree with this.

**Comment by joshsmith** [comment]
I think whomever makes a change, part of their responsibility in that PR should be adding to the changelog.

**Comment by venkatd** [comment]
How do these get approved?

**Comment by venkatd** [comment]
I think we can approve these informally for now. No need for any ceremony here.

**Comment by joshsmith** [comment]
Yeah, I agree. Just move it to `Needs Pull Request` if we generally agree it's a good idea, then leave it open until a PR closes it out.
